project_euler/problem_11.py

```diff
@@ -0,0 +1,99 @@
+# https://projecteuler.net/problem=11
+
+grid = [[8, 2, 22, 97, 38, 15, 0, 40, 0, 75, 4, 5, 07, 78, 52, 12, 50, 77, 91, 8],
```

Comment:
It doesn't matter much for this problem, but `07` is actually an octal number in Python2. Consider:
```
>>> 07
7
>>> 010
8
```